packages/kbn-i18n/src/core/i18n.ts

```diff
@@ -24,10 +24,17 @@ import IntlRelativeFormat from 'intl-relativeformat';
 import { Messages, PlainMessages } from '../messages';
 import { Formats, formats as EN_FORMATS } from './formats';
 import { hasValues, isObject, isString, mergeAll } from './helper';
+import { charMap } from './pseudo_locale';
 
 // Add all locale data to `IntlMessageFormat`.
 import './locales.js';
 
+/**
+ * Matches every single [A-Za-z] character, `<tag attr="any > text">` and `](markdown-link-address)`
+ */
+const CHARS_FOR_PSEUDO_LOCALIZATION_REGEX = /[A-Za-z]|(\]\([\s\S]*?\))|(<([^"<>]|("[^"]*?"))*?>)/g;
```

Comment:
__issue:__ let's move all pseudo locale specific code to `./pseudo_locale.ts` so that it's well contained and expose only `isPseudoLocale(locale: string): boolean` and `translateUsingPseudoLocale(message: string): string` that will be used here and in react provider. It's internal feature and we shouldn't expose on the core `i18n`.